### ISSUE TITLE
Is there a way to get the state of an audio source?

### ISSUE THREAD

**Issue by Plushfire5** [nuttylmao/NOOBS-CMDR#18]
Hello,

I'd like to know, if instead of turning off and on an audio source, e.g. Microphone, you could get the state it is currently in (muted/unmuted) ?
I'd like to be able to press a key (using HID Macros), and through an AHK script run a VBS that gets the state in a variable let's say, and run a specific line depending on the state
e.g.
```
If mic == muted
{
SoundPlay, umute.wav
Run, unmute_mic.vbs
}
else if mic == unmuted
{
SoundPlay, mute.wav
Run, mute_mic.vbs
}
```

I haven't figured out yet how I will do the variable saving in AHK yet but I'll investigate it later.
So basically I'd like to know the state of a source, is that possible?
